### DISCUSSION TITLE
Is there any way to cast/mutate query results before they are returned?

### DISCUSSION THREAD

**Discussion by maxloh-prizm** [prisma/prisma#7275] [Q&A] [answered]
Hi. I am looking for a way to cast/mutate query results before they are returned by `find*` functions, like [mutator in Laravel](https://laravel.com/docs/8.x/eloquent-mutators).

Is that possible with Prisma?

**Comment by ryands17** [comment, marked as answer]
@maxloh-prizm 👋 

Currently we do not support mutating query results, but we do have a request for this where we will introduce a class based approach that would fit your use case. It would be great if you could add a 👍 to the request so that we can look at the priority for this.

https://github.com/prisma/prisma/issues/5998
